src/main/java/ca/bc/gov/iamp/bcparis/api/exception/ExceptionHandlerController.java

```diff
@@ -20,6 +20,7 @@
 	@ResponseStatus(HttpStatus.BAD_REQUEST)
 	@ExceptionHandler({ InvalidMessageType.class })
 	public Error invalidMessage(InvalidMessageType e) {
+		e.printStackTrace();
```

Comment:
@magostta  Change all e.printStackTrace to log.error. printStackTrace won't pass Sonar quality